message.go

```diff
@@ -14,5 +21,16 @@ type Message struct {
 }
 
 func (m Message) String() string {
-	return fmt.Sprintf("%s\t%s\t%s\t%s", m.Timestamp.Format(time.RFC3339), m.Level, m.Labels, m.Message)
+	var Color string
+	switch m.Level {
+	case "debug":
+		Color = DebugColor
+	case "warning":
+		Color = WarningColor
+	case "error":
+		Color = ErrorColor
+	default:
```

Comment:
This would be used for "trace" and "info", although in practice I don't think Arca ever uses "trace". I think I'd prefer explicitly handling "info" with a default case for "anything else that gets through"...

Reply:
I can do that, but it will be redundant code since the idea here is for `info` level logs to use the regular terminal font, so it uses the ANSI reset code rather than an explicit color setting, and the reset code would also be used as the default case.

Reply:
Sometimes, redundant code is clearer, and easier to maintain. Especially since "info" is probably the most common, and the fact that it's missing from your list is likely to trigger new readers.

I was also confused by your choice of "reset" as the name. And, OK, I get that you're using the "reset" ANSI sequence; although, in the context you're using it, if there's anything to "reset" then the code is probably broken. Your "warning" and "error" constants, for example, aren't named by the ANSI sequence (or even the specific color), and I prefer that style ... the fact that your default happens to be the ANSI reset sequence is more or less irrelevant, and I think the empty string ought to work just as well ... but a name like `DefaultColor` or `NormalColor` would be more consistent with the others.

And, personally, I'd be happy to see `InfoColor` and `DefaultColor` even if they both use the reset sequence now... just because that'd clarify the intent.